Gems/EditorPythonBindings/Code/Tests/PythonThreadingTests.cpp

```diff
@@ -217,4 +217,79 @@ namespace UnitTest
         e.Deactivate();
     }
 
+    TEST_F(PythonThreadingTest, PythonInterface_DebugTrace_CallsOnTick)
+    {
+        enum class LogTypes
+        {
+            Skip = 0,
+            OnPrewarning
+        };
+
+        m_testSink.m_evaluateMessage = [](const char* window, const char* message) -> int
+        {
+            if (AzFramework::StringFunc::Equal(window, "python"))
+            {
+                if (AzFramework::StringFunc::StartsWith(message, "OnPrewarning"))
+                {
+                    return aznumeric_cast<int>(LogTypes::OnPrewarning);
+                }
+            }
+            return aznumeric_cast<int>(LogTypes::Skip);
+        };
+
+        AZ::Entity e;
```

Comment:
nitpick - could have a better name here than e.

Reply:
Yeah... I copy pasted the same skeleton from the previous 2 tests. I can put a better name in a subsequent PR.

Reply:
👍